src/ansys/sherlock/core/parts.py

```diff
@@ -181,9 +181,9 @@ def update_parts_list(
             Name of the CCA.
         part_library : str
             Name of the parts library.
-        matching : UpdatesPartsListRequestMatchingMode
+        matching : PartsListSearchMatchingMode
             Matching mode for updates.
-        duplication : UpdatesPartsListRequestDuplicationMode
+        duplication : PartsListSearchDuplicationMode

```

Comment:
For consistency, would be better to rename these to matching_mode and duplication_mode.